README.md

```diff
@@ -16,15 +14,14 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://js-vite-template.netlify.app/)
+[Live Demo Link](adels-react-calculator-app.netlify.app)
 
 ### Development (Running locally)
 
 - Clone the project
 
 ```bash
-git clone https://github.com/bolah2009/js-vite-template.git
-
+git clone git@github.com:Yinkusadel/Adels-react-calculator-app.git
```

Comment:
```suggestion
git clone git@github.com:Yinkusadel/react-calculator.git
```

I believe this would be the best it's more straightforward with no redundant naming.